### PR TITLE
101: Client checks for new transactions

### LINKED ISSUE #101: Client checks for new transactions
**Description:**

Currently, a client has no way to know that a transaction has been sent to it. Implement a way for the client to check or to be notified if they have been sent a transaction and update their state as such. 

- Figure out a way to have the client get notified when they receive a transaction 

**Acceptance Criteria:**

- [x] When the client is sent a transaction, it receives that transaction and updates its state accordingly with the new transaction

### PR DESCRIPTION
**Pull request by ZackHolmberg**
closes #101

### CODE REVIEW

**Comment by abhisachdev17** [comment]
Did you fix the wrong mining toast with this or?

**Review by madisonfines — APPROVED** [review]
nailed it

File: src/frontend/src/App.vue [hunk 1]
```diff
@@ -81,9 +80,8 @@ export default class App extends Vue {
       });
 
       socket.on("reward", () => {
+          this.$store.dispatch("ACTION_FETCH_TRANSACTION_HISTORY");
           this.$store.dispatch("ACTION_FETCH_WALLET_AMOUNT");
-          this.$store.dispatch("ACTION_DISPLAY_TOAST", { message: 'Mining reward received!', type: 'success' })
-
```
Comment (abhisachdev17) [inline src/frontend/src/App.vue:86]: Do we not want a toast message for mining rewards?

**Review by abhisachdev17 — APPROVED** [review]
YOLO

**Pull request merged**